src/selectfield/selectfield.js

```diff
@@ -167,6 +167,11 @@
         this.select_.addEventListener('focus', this.boundFocusHandler);
         this.select_.addEventListener('blur', this.boundBlurHandler);
         this.select_.addEventListener('reset', this.boundResetHandler);
+        
+        var icon = document.createElement('div');
+        icon.className = 'mdl-selectfield__icon';
```

Comment:
Yea, this could go either way. I at least got rid of the need for an event handler. I like explicit markup just felt it would be redundant in this case. It's your component so I will make the adjustment.